server/game/cards/05-DT/Saurarium.js

```diff
@@ -0,0 +1,20 @@
+const Card = require('../../Card.js');
+
+class Saurarium extends Card {
+    //Each creature with the lowest power cannot reap.
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            targetController: 'any',
+            match: (card, context) => {
+                let lowestPower = context.game.creaturesInPlay.sort((a, b) => a.power - b.power)[0]
```

Comment:
if there are no creatures in play, this may cause an "undefined" exception (undefined.power)